asteroid.js

```diff
@@ -10,23 +15,26 @@ function Asteroid(pos, r) {
     this.r = random(15, 50);
   }
 
-
   this.vel = p5.Vector.random2D();
   this.total = floor(random(5, 15));
   this.offset = [];
   for (var i = 0; i < this.total; i++) {
     this.offset[i] = random(-this.r * 0.5, this.r * 0.5);
   }
+  this.rot = 0;
+  this.rotSpeed = random(-0.05, 0.05);
 
   this.update = function() {
     this.pos.add(this.vel);
+    this.rot += this.rotSpeed;
   }
 
   this.render = function() {
     push();
     stroke(255);
     noFill();
     translate(this.pos.x, this.pos.y);
+    rotate(this.rot);
```

Comment:
Are these changes just indentation or something else, since these lines seem to already be on the gh-pages branch.